src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -388,6 +512,7 @@ populate_allowed_fds( void * scratch,
   out_fds[ out_cnt++ ] = init_ctx->xsk->xsk_fd;
   if( FD_UNLIKELY( init_ctx->lo_xsk ) )
     out_fds[ out_cnt++ ] = init_ctx->lo_xsk->xsk_fd;
+  out_fds[ out_cnt++ ] = fd_ip_netlink_get(init_ctx->ip)->fd;
```

Comment:
another style nit (won't keep making this comment, but it's the style throughout these files at least)

```suggestion
  out_fds[ out_cnt++ ] = fd_ip_netlink_get( init_ctx->ip )->fd;
```